Sources/PackageRegistry/RegistryClient.swift

```diff
@@ -22,10 +22,10 @@ import struct TSCUtility.ZipArchiver
 
```

Comment:
Are the changes in this file mainly around error handling?

Reply:
yes, the diff is a bit crazy because if indentation changes but its all error handling